File: miltecti-api/Models/AnuncioModel.cs
using Microsoft.AspNetCore.Mvc.ModelBinding;

namespace miltecti_api.Models
{
    public class AnuncioModel
    {
        public int Id { get; set; }
        public string NomeAnuncio { get; set; }
        public DateTime DataPublicacao { get; set; }
        public decimal Valor { get; set; }
        public string Cidade { get; set; }
        public string? Categoria { get; set; }
        public string? Modelo { get; set; }
        public string? Condicao { get; set; }
        public int? Quantidade { get; set; }
        public string? TipoServico { get; set; }
        public string TipoAnuncio { get; set; } // "Produto" ou "Serviço"
    }
}

File: miltecti-angular/miltecti-frontend/src/app/Models/anuncio.model.ts
export interface Anuncio {
    id: number;
    nomeAnuncio: string;
    dataPublicacao: string;
    valor: number;
    cidade: string;
    categoria?: string;  
    modelo?: string;     
    condicao?: string;   
    quantidade?: number; 
    tipoServico?: string; 
    tipoAnuncio: string; 
  }
  